modules/making-open-source-more-inclusive.adoc

```diff
@@ -0,0 +1,9 @@
+// This module is included in the following assemblies:
```

Comment:
You needed to create this module because it does not exist in a repo?

This module does already exist in the `openshift-docs` repo, so maybe reuse is posssible?

Reply:
I cannot reuse because `build-docs` have a separate repo.